app/upgrades/v19/constants.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/CosmosContracts/juno/v19/app/upgrades"
 )
 
+const (
+	Core1MultisigVestingAccount = "juno190g5j8aszqhvtg7cprmev8xcxs6csra7xnk3n3"
+	CharterCouncil              = "juno1nmezpepv3lx45mndyctz2lzqxa6d9xzd2xumkxf7a6r4nxt0y95qypm6c0"
```

Comment:
I have double checked these addresses and they are correct. See for yourself:
- https://daodao.zone/dao/juno1nmezpepv3lx45mndyctz2lzqxa6d9xzd2xumkxf7a6r4nxt0y95qypm6c0/proposals
- https://www.mintscan.io/juno/address/juno190g5j8aszqhvtg7cprmev8xcxs6csra7xnk3n3

Reply:
Triple checked!